Anytype/Sources/PresentationLayer/Common/SwiftUI/Buttons/StandardButton.swift

```diff
@@ -4,37 +4,45 @@ import AnytypeCore
 typealias StandardButtonAction = () -> Void
 
 struct StandardButton: View {
-    let text: String
+    enum Content {
+        case text(String)
+        case image(ImageAsset)
+    }
+    
+    let content: Content
     let info: String?
     let inProgress: Bool
     let style: StandardButtonStyle
     let action: StandardButtonAction
+    let corners: UIRectCorner
     let holdPressState: Bool // Use only for demo
     
     @State private var isPressed: Bool = false
     @Environment(\.isEnabled) private var isEnable
     
     init(
-        _ text: String,
+        _ content: Content,
```

Comment:
You can create a two convenience init with text and image. Then don't needs to refactoring everywhere